tests/testthat/_snaps/pivot-wide.md

```diff
@@ -162,9 +162,9 @@
       * Use `values_fn = {summary_fun}` to summarise duplicates.
       * Use the following dplyr code to identify duplicates.
         {data} %>%
-          dplyr::group_by(a, key) %>%
-          dplyr::summarise(n = dplyr::n(), .groups = "drop") %>%
-          dplyr::filter(n > 1L)
+        dplyr::group_by(a, key) %>%
+        dplyr::summarise(n = dplyr::n(), .groups = "drop") %>%
+        dplyr::filter(n > 1L)
```

Comment:
I think the tabbing is off? i.e. `{data}` shouldn't be aligned with the following lines right?

Reply:
Right, but I gave up getting cli to not re-indent.

Reply:
Oh that is unfortunate